src/redux/rocketReducer/RocketReducer.js

```diff
@@ -0,0 +1,27 @@
+import { RocketConstants } from './RocketAction';
+
+const initialState = {
+  rockets: [],
+};
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Kindly remove disabled Linters, Linting improves readability, removes errors before execution and also make your code more professional ✊ 

Please check for other occurences.